src/components/Editor/Editor.tsx

```diff
@@ -106,6 +109,21 @@ export const Editor: FunctionComponent<EditorProps> = props => {
         navigator.clipboard.writeText(content);
     }, [content]);
 
+    const onLinkClick = useCallback(() => {
+        navigator.clipboard.writeText(content).then(() => {
```

Comment:
I think, now `clipboard.writeText` is not important anymore. Please, remove `navigator.clipboard.writeText(content)` and use `pushFileContent` only.